addons/web/static/tests/views/kanban/kanban_view_tests.js

```diff
@@ -9442,6 +9442,49 @@ QUnit.module("Views", (hooks) => {
         ]);
     });
 
+    QUnit.test("Column(s) progressbar filters working functionality", async (assert) => {
```

Comment:
This test doesn't reproduce the faulty scenario at all. As a matter of fact it already pass without the fix.

Reply:
Hello @aab-odoo 

I am sorry for the delay but my issue arises only when, once the whole kanban view is rendered then after that if we create two or more new stages, in those stages we create new tasks or ticket, and without opening the tasks or tickets if we assign them different states associated with a colour in progressbar and then click on any of the colour in the progressbar in any of the newly created stages.

- My test case is not working because in it i do not create new stages once the view is rendered.

- Since the fix was in web i tried to write the test case in the web. And while writing the test case i took the reference of previously written test cases and data. And i am facing two issues
> 1) While making the view and including the progressbar if my groupby field is not of many2one type(or any other realtional type, only used for stages ) then the progressbar would have have colours filled in the progressbar but would not have a button in the view to create new stages  For eg: - 
https://github.com/odoo/odoo/blob/47ede112ed542239aa4f73aee8301f84f1098e55/addons/web/static/tests/views/kanban/kanban_view_tests.js#L5411-L5420

>2) While making the view and including the progressbar if my groupby field is of type many2one (or any other realtional type, only used for stages ) then the progressbar would not have colours filled in it but would have a button in the view to create new stages  For eg: - 
https://github.com/odoo/odoo/blob/47ede112ed542239aa4f73aee8301f84f1098e55/addons/web/static/tests/views/kanban/kanban_view_tests.js#L12760-L12770

- I tried to include `quick_create` in the view with the progressbar but still could not achieve the desired flow. 
- I hope i am not missing out anything and since you have already written tests for the view i request you to please guide me where i am mistaken so that i can write the appropriate test which goes with the usual flow !

Thanks